src/currency.h

```diff
@@ -68,6 +68,7 @@ enum Currencies {
 	CURRENCY_MYR,       ///< Malaysian Ringgit
 	CURRENCY_LVL,       ///< Latvian Lats
 	CURRENCY_PTE,       ///< Portuguese Escudo
+        CURRENCY_UAH,       ///< Ukrainian Hryvnia
```

Comment:
Indent with tabs, not spaces.

```suggestion
	CURRENCY_UAH,       ///< Ukrainian Hryvnia
```